WordPress/Classes/ViewRelated/Reader/ReaderStrings.swift

```diff
@@ -0,0 +1,6 @@
+/// Encapsulates user facing strings in Reader
+extension String {
+    static var savedForLaterMenuTitle: String {
```

Comment:
Sorry for being awkward again, but I'm not a huge fan of public extensions to classes like `String` here, to add a very specific property that's only used in one place in the app. 

I think I'd prefer a private enum in the view model (could be called `Strings`), with static constants for any strings we need. It's the model we follow in a lot of places in the app at the moment, and it keeps the constants much closer to where we actually need them. Would you be open to that?

Reply:
I agree. This was meant to be just the starting point, and meant to grow with more strings as we go. I'll move it to the view model

Reply:
I made it a struct though. I am not a huge fan of scattering calls to NSLocalizedString, which add a lot of noise. An enum forces "raw values for enum case must be a literal", and to be honest, adding a method to the enum just to wrap the raw value in a call to NSLocalizedString seems a bit too much.